test/transport_test.js

```diff
@@ -0,0 +1,61 @@
+'use strict';
+const expect = require('chai').expect;
+const mockery = require('mockery');
+const sinon = require('sinon');
+const Events = require('events');
+
+describe('Test Transport.js',()=>{
+    const wsConstructorSpy = sinon.spy();
+    class Websocket extends Events{
+        constructor(conf){
+            super();
+            wsConstructorSpy();
+        }
+        terminate(){
+
+        }
+    }
+    mockery.enable({
+        warnOnReplace: false,
+        warnOnUnregistered: false,
+        useCleanCache: true
+    });
+    mockery.registerMock('ws', Websocket);
+    const Transport = require('./../lib/Transport');
+    let transport;
+    beforeEach(()=>{
+        wsConstructorSpy.reset();
+        transport = new Transport({});
+    });
```

Comment:
Please add blank lines between all these blocks to match our other files.

Reply:
ok